storage/src/vespa/storage/storageserver/mergethrottler.cpp

```diff
@@ -69,6 +69,7 @@ MergeThrottler::Metrics::Metrics(metrics::MetricSet* owner)
       active_window_size("active_window_size", {}, "Number of merges active within the pending window size", this),
       estimated_merge_memory_usage("estimated_merge_memory_usage", {}, "An estimated upper bound of the "
                                    "memory usage of the merges currently in the active window", this),
+      merge_memory_limit("merge_memory_limit", {}, "The active soft limit for memory used by merge operations on this node", this),
```

Comment:
Please add in the comment that this is in bytes.

Reply:
Addressed in #29205